src/frontends/onnx/frontend/src/core/tensor.hpp

```diff
@@ -157,14 +159,19 @@ class Tensor {
             return ov::element::u64;
         case TensorProto_DataType::TensorProto_DataType_BFLOAT16:
             return ov::element::bf16;
+        case TensorProto_DataType::TensorProto_DataType_FLOAT8E4M3FN:
+            return ov::element::f8e4m3;
+        case TensorProto_DataType::TensorProto_DataType_FLOAT8E5M2:
```

Comment:
Add `f8e8m0` also?

Reply:
As I mentioned - I don't see ONNX supports it, even we do.